website/plugins/proto-messages/index.js

```diff
@@ -151,7 +151,56 @@ module.exports = () => {
           } else {
             // Otherwise, push the line normally.
             const isNotLast = i + 1 < length;
-            let val = `${line}`;
+            const matches = getLinksFromALine(line);
+
+            if (matches.length) {
+              const elements = [];
```

Comment:
This is Element? https://developer.mozilla.org/en-US/docs/Web/API/Element

Reply:
Ah, yes, it was a poor variable name. A more fitting name would be `hastElements`, perhaps. https://github.com/syntax-tree/hast#element

Reply:
Actually, I decided to remove it. We can push to `children` right away.